frontend/js/src/settings/troi/SelectTroiPreferences.tsx

```diff
@@ -96,14 +96,14 @@ class SelectTroiPreferences extends React.Component<
         <Helmet>
           <title>Select Playlist Preferences</title>
         </Helmet>
-        <h3>Configure auto export of generated playlists</h3>
+        <h3>Playlist auto-export</h3>
         <p>
-          If this setting is turned on, ListenBrainz will automatically export
-          your generated playlists (Weekly Jams, Weekly Exploration, ...) to
-          Spotify everyday.
+          If auto-export is turned on, ListenBrainz will automatically export
+          your generated playlists (Weekly Jams, Weekly Exploration, etc) to
+          Spotify, daily.
```

Comment:
I wonder if the wording was already wrong here, as the weekly playlists are exported, well, weekly and not daily.
Maybe just remove that last word?

Reply:
Done, thanks!